tutorials/getting_started/predicting_paper_venues/predicting_paper_venues_pt2.md

```diff
@@ -71,13 +71,7 @@ In this example we also would like to return the list of all possible labels
 We first get the mapping from indices to labels, and then we convert
 it to a list where position 0 is label 0, and so on.
 
-`predict_json` returns a JSON dict, where the first element is
-the `Instance` and the second element is a `dict` that the elements
-of `Model.forward_on_instance` will be added to. Anything that we want
-in our JSON output that's not produced by `forward()` goes in it.
-
-Here that's just the list of `all_labels`, so that's what we put in the `dict`.
-If we didn't need that, we'd just use an empty `dict` there.
+`predict_json` returns a JSON dict with keys `"instance"` and `"all_labels"`.
```

Comment:
Sorry, thinking more carefully now, we should probably be clear that the choice of keys is arbitrary. How about the following:

```
In a typical `Predictor` `predict_json` returns the result of `Model.forward_on_instance`, but you can return an arbitrary JSON dict. In this case we simply put the predicted instance under `"instance"` and the labels under `"all_labels"`.
```

Reply:
Updated.